src/screens/Signup/state.ts

```diff
@@ -262,6 +261,11 @@ export function useSubmitSignup({
           inviteCode: state.inviteCode.trim(),
           verificationCode: verificationCode,
         })
+        /*
+         * Must happen last so that if the user has multiple tabs open and
+         * createAccount fails, one tab is not stuck in onboarding — Eric
+         */
+        onboardingDispatch({type: 'start'})
       } catch (e: any) {
         onboardingDispatch({type: 'skip'}) // undo starting the onboard
```

Comment:
Seems like we don't want this now then? Cause it might also change onboarding state — but we haven't actually started changing it.

Reply:
Good question! This dispatch actually just sets onboarding state to `Home`, which is the only value that [results in onboarding being inactive.](https://github.com/bluesky-social/social-app/blob/main/src/state/shell/onboarding.tsx#L120) This is also the default value for new accounts.

So it's not a problem, but there's also no reason to leave this in here since it won't have any effect anymore. I'll remove 👍